library/std/src/sys/pal/zkvm/alloc.rs

```diff
@@ -5,7 +5,9 @@ use crate::alloc::{GlobalAlloc, Layout, System};
 unsafe impl GlobalAlloc for System {
     #[inline]
     unsafe fn alloc(&self, layout: Layout) -> *mut u8 {
-        abi::sys_alloc_aligned(layout.size(), layout.align())
+        unsafe{
```

Comment:
rustfmt is going to ask for this
```suggestion
        unsafe {
```

Reply:
actually rustfmt expects the entire thing to be on one line lol

Reply:
yeah, sorry about that - I should have checked for this earlier